BEIMA.Backend/host.json

```diff
@@ -7,5 +7,10 @@
                 "excludedTypes": "Request"
             }
         }
+    },
+    "extensions": {
+        "http": {
+          "routePrefix": "beima"
```

Comment:
Could the functional tests be failing because of this added prefix?

Reply:
Yeah, I think you need to add the correct url prefix in the FT BeimaClient.cs, and the FT base, I can branch from this and do that real quick if needed?

Reply:
You'll likely get it done faster than I will